gm4_area_markers/data/area_markers/functions/recipe_check.mcfunction

```diff
@@ -0,0 +1,9 @@
+#run from #custom_crafters:recipe_check
+#@s = custom crafter entity
+
+#craft tier 1
+execute if score @s gm4_slot_count matches 4 if block ~ ~ ~ dropper{Items:[{Slot:1b,id:"minecraft:name_tag"},{Slot:3b,id:"minecraft:iron_ingot"},{Slot:5b,id:"minecraft:iron_ingot"},{Slot:7b,id:"minecraft:name_tag"}]} run data merge block ~ ~ ~ {Items:[{Slot:8,id:"player_head",Count:1b,tag:{gm4_custom_crafters:{multiplier:1},gm4_area_marker:1b,gm4_am_tier:1b,display:{Name:"{\"translate\":\"%1$s%3427655$s\",\"with\":[\"Area Marker Tier 1\",{\"translate\":\"item.gm4.area_markers.tier_1\"}],\"italic\":\"false\"}"},SkullOwner:{Id:"9b36c3fe-187f-4686-b472-4c7d7670804a",Properties:{textures:[{Value:"eyJ0ZXh0dXJlcyI6eyJTS0lOIjp7InVybCI6Imh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvNTJiNmZiMGRlYzgwMmJlMmJiYTQ4ODg2MDA5MDlkZGIxMzVhMGNlYjMzZWFmN2M0ZjI1NTNmNTE4Nzc2ZDlkZSJ9fX0="}]}}}}]}
+#craft tier 2
+execute if score @s gm4_slot_count matches 4 if block ~ ~ ~ dropper{Items:[{Slot:1b,id:"minecraft:name_tag"},{Slot:3b,id:"minecraft:gold_ingot"},{Slot:5b,id:"minecraft:gold_ingot"},{Slot:7b,id:"minecraft:name_tag"}]} run data merge block ~ ~ ~ {Items:[{Slot:8,id:"player_head",Count:1b,tag:{gm4_custom_crafters:{multiplier:1},gm4_area_marker:1b,gm4_am_tier:2b,display:{Name:"{\"translate\":\"%1$s%3427655$s\",\"with\":[\"Area Marker Tier 2\",{\"translate\":\"item.gm4.area_markers.tier_2\"}],\"italic\":\"false\"}"},SkullOwner:{Id:"68960c41-2c50-4ba4-8de0-665bae643405",Properties:{textures:[{Value:"eyJ0ZXh0dXJlcyI6eyJTS0lOIjp7InVybCI6Imh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvNmE3NzZlZjA0OTAxZDRkN2Y2Y2ZkMGU5MjdkNjQwNDFhY2M4NWUyNmRhYjU5MzZmODQ4Yjk2NTViNWQyNTdkZiJ9fX0="}]}}}}]}

```

Comment:
I can't think if any alternate recipes either, but these don't seem to make sense lore wise; as in, why would a diamond increase the range?

Reply:
I haven't an idea either